src/shogun/machine/LinearMachine.h

```diff
@@ -66,12 +66,26 @@ class CLinearMachine : public CMachine
 		/** default constructor */
 		CLinearMachine();
 
+		/**constructor with bias flag */
+		CLinearMachine(bool flag);
```

Comment:
proper naming of the argument please. And please also doxygen document properly. Use CAPITAL letters in sentence starts, use @param to document parameters